app/views/course/admin/assessment_settings/edit.html.slim

```diff
@@ -0,0 +1,53 @@
+= page_header
+
+h2
+  span = t('.categories_and_tabs')
+  div.pull-right
+    - if can?(:create, Course::Assessment::Category)
+      = new_button([current_course, :admin, :assessments, :category])
+
+= simple_form_for(current_course, url: course_admin_assessments_path(current_course)) do |f|
+  table.table.table-hover.category-items
+    thead
+      th
+      th
+      th = t('.title')
+      th = t('.weight')
+      th = t('.parent-category')
+      th
+    tbody
+      = f.simple_fields_for(:assessment_categories, @categories) do |category_form|
+        - category = category_form.object
+        = content_tag_for(:tr, category) do
+          td
+            span = fa_icon :reorder
+          td
+          td = category_form.input :title, label: false, input_html: { id: 'category_title' }
```

Comment:
It works because I'm using a selector to filter by content tag first, but my bad, should have been a class.